packages/wallet-sdk/src/util/encoding.ts

```diff
@@ -0,0 +1,55 @@
+import { hexToBytes, numberToHex, trim } from "viem";
+
+export function base64ToBase64Url(base64: string): string {
+  return base64.replaceAll('+', '-').replaceAll('/', '_').replace(/=+$/, '');
+}
+
+export function arrayBufferToBase64Url(buffer: ArrayBuffer): string {
+  // First convert to regular base64
+  const base64 = Buffer.from(buffer).toString('base64');
```

Comment:
I don't think we should be using Buffer directly. The SDK needs to run in web context without the need to polyfill.

Reply:
```tsx
function arrayBufferToBase64(buffer) {
  let binary = '';
  const bytes = new Uint8Array(buffer);
  const len = bytes.byteLength;
  for (let i = 0; i < len; i++) {
    binary += String.fromCharCode(bytes[i]);
  }
  return btoa(binary);
}
```

https://stackoverflow.com/questions/9267899/how-can-i-convert-an-arraybuffer-to-a-base64-encoded-string

Reply:
gotcha, should probably be a linter warning then. will look into that

Reply:
Downside of biome. Still some lacking tooling. hmm...